### DISCUSSION TITLE
How to implementation A field add B field greater than a number in query where condition?

### DISCUSSION THREAD

**Discussion by nanlong** [SeaQL/sea-orm#862] [Q&A] [answered]
How to implementation A field add B field greater than a number in query where condition?

**Comment by tyt2y3** [comment, marked as answer]
You'd need to get the underlying query builder first, https://docs.rs/sea-orm/latest/sea_orm/query/trait.QuerySelect.html#tymethod.query and https://docs.rs/sea-query/latest/sea_query/expr/struct.Expr.html#method.add
